librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -48,6 +48,29 @@ commands:
             type: RZ_CMD_ARG_TYPE_NUM
           - name: h
             type: RZ_CMD_ARG_TYPE_NUM
+  - name: ph
+    summary: Print hash/message digest or entropy
+    cname: cmd_print_default
+    subcommands:
+      - name: ph
+        summary: Prints a hash/message digest or entropy
+        cname: cmd_print_msg_digest
+        args:
+          - name: algo
+            type: RZ_CMD_ARG_TYPE_STRING
+          - name: length
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
```

Comment:
In theory we could eliminate these kind of things and just rely on `@!`, which works for every command. If we use that approach we can assume all commands work on core->offset and with core->blocksize, but you can always use `@` and `@!` to change those. Just an idea anyway...

Reply:
i agree. i thought about that too.. but usability may sinks..